plugins/com.google.cloud.tools.eclipse.appengine.newproject.test/src/com/google/cloud/tools/eclipse/appengine/newproject/StandardProjectWizardTest.java

```diff
@@ -0,0 +1,15 @@
+package com.google.cloud.tools.eclipse.appengine.newproject;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StandardProjectWizardTest {
+
+  private StandardProjectWizard wizard = new StandardProjectWizard();
+
+  @Test
+  public void testCanFinish() {
+    Assert.assertTrue(wizard.canFinish());
```

Comment:
This seems weird: I wouldn't expect that a new wizard wouldn't be finishable until its project name had been entered